### PR TITLE
Translated into russian file: formatting.po

### PR DESCRIPTION
**Pull request by Cub11k**
## Description
Include changes, new features and etc:

## Describe your tests
How did you test your change?

Python version:

OS:

## Checklist:
- [ ] I added/edited example on new feature/change (if exists)
- [ ] My changes won't break backward compatibility
- [ ] I made changes both for sync and async

### CODE REVIEW

File: docs/source/locales/ru/LC_MESSAGES/formatting.po [hunk 1]
```diff
@@ -20,23 +20,23 @@ msgstr ""
...
 
 #: of telebot.formatting:1
 msgid "Markdown & HTML formatting functions."
-msgstr ""
+msgstr "Функции форматирования Markdown & HTML."
 
 #: of telebot.formatting.escape_html:1
 msgid "Escapes HTML characters in a string of HTML."
-msgstr ""
+msgstr "Экранирует HTML символы в HTML строке."
```
Comment (coder2020official) [inline docs/source/locales/ru/LC_MESSAGES/formatting.po:39]: this doesn't sound good.. what about making it "пропускает" ?

File: docs/source/locales/ru/LC_MESSAGES/formatting.po [hunk 2]
```diff
@@ -147,105 +147,105 @@ msgstr ""
 #: telebot.formatting.mspoiler:6 telebot.formatting.mstrikethrough:6
 #: telebot.formatting.munderline:6
 msgid "True if you need to escape special characters. Defaults to True."
-msgstr ""
+msgstr "True если вам нужно экранировать спец. символы. По умолчанию True."
```
Comment (coder2020official) [inline docs/source/locales/ru/LC_MESSAGES/formatting.po:150]: you can use "избегать" here i think

File: docs/source/locales/ru/LC_MESSAGES/formatting.po (hunk 1 above)
Comment (Cub11k) [reply docs/source/locales/ru/LC_MESSAGES/formatting.po:39]: Agreed. it doesn't sound good, but the thing is that this function doesn't really skip("пропускает") HTML characters. It replaces them with their codes, making them non-HTML for parser.

File: docs/source/locales/ru/LC_MESSAGES/formatting.po (hunk 2 above)
Comment (Cub11k) [reply docs/source/locales/ru/LC_MESSAGES/formatting.po:150]: Same as my respond for previous comment. It doesn't really skip or avoid characters, it does replace them with their codes, making them non-HTML for parser.

File: docs/source/locales/ru/LC_MESSAGES/formatting.po (hunk 1 above)
Comment (Cub11k) [reply docs/source/locales/ru/LC_MESSAGES/formatting.po:39]: Might be convinient enough to use "пропускает" cause parser will skip escaped characters. Will make an update

**Review by coder2020official — APPROVED** [review]

**Comment by coder2020official** [comment]
LGTM, thanks

**Pull request merged**